lib/venue.rb

```diff
@@ -1,2 +1,38 @@
 class Venue
+  attr_accessor :patrons
+  attr_reader :name, :capacity
+  def initialize(name, int)
+    @name = name
+    @capacity = int
+    @patrons = []
+  end
+
+  def add_patron(i)
+    self.patrons << i.to_s
```

Comment:
You wouldn't need to use `.to_s` here because it is already coming in as a string